.tekton/external-dns-container-ext-dns-optr-1-3-rhel-9-pull-request.yaml

```
@@ -212,6 +212,7 @@ spec:
       - name: BUILD_ARGS
         value:
         - $(params.build-args[*])
+        - 'FULL_COMMIT=$(tasks.clone-repository.results.short-commit)'
```

```suggestion
        - 'FULL_COMMIT=$(tasks.clone-repository.results.commit)'
```

Just noticed that we made the same mistake in the operator repository.